src/plugins/files/server/routes/file_kind/update.ts

```diff
@@ -24,7 +24,11 @@ const rt = {
     meta: schema.maybe(commonSchemas.fileMeta),
   }),
   params: schema.object({
-    id: schema.string(),
+    id: schema.string({
```

Comment:
Mixing code-related changes with docs changes are risky close to a FF.

Reply:
Yeah this was not meant to be comitted